src/setup.js

```diff
@@ -0,0 +1,30 @@
+const { sequelize } = require('./utils/db');
+const { User } = require('./models/user.model');
+
+require('./models/token.model');
+require('./models/message.model');
+
+const { Room } = require('./models/room.model');
+
+(async () => {
+  try {
+    await sequelize.sync({ alter: true });
+
+    const firstUserData = {
+      name: 'Bodya',
+      email: 'test@gmail.com',
+      password: '$2b$09$vXUnnK7qwX9keDAintwqd.LEaRanah6c1xOaTR7X2TAAv.vyZ3uO6',
```

Comment:
Ensure that the hardcoded password is securely handled and not used in production environments. Consider using environment variables or a secure vault for sensitive data.